src/runtime_src/core/include/xrt.h

```diff
@@ -1215,6 +1215,21 @@ size_t
 xclDebugReadIPStatus(xclDeviceHandle handle, enum xclDebugReadType type,
                      void* debugResults);
 
+/**
+ * xclStoreAppContext() - Store application context
+ *
+ * @handle:        Device handle
+ */
+XCL_DRIVER_DLLESPEC int xclStoreAppContext(xclDeviceHandle handle);
+
+/**
+ * xclDeleteAppContext() - Delete application context
+ *
+ * @handle:        Device handle
+ */
+XCL_DRIVER_DLLESPEC int xclDeleteAppContext(xclDeviceHandle handle);
+
+
```

Comment:
Not enough documentation.  What is this?
Also, please follow the formatting convention in this file and keep all function declarations to the left.